### PR TITLE
Alignmnent issues 

### PR DESCRIPTION
**Pull request by SrinandhiniM**
Alignmnent issues posted  are resolved

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -1,57 +1,47 @@
...
+      <%= f.label :last_name %>
+      <%= f.text_field :last_name, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+      <%= f.password_field :password, autocomplete: "new-password", class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control" %>
```
Comment (iMagesh) [inline app/views/devise/registrations/edit.html.erb:30]: this code seems to be duplicated in the below file as well. Can you refactor it?

**Review by iMagesh — CHANGES_REQUESTED** [review]
Just need to refactor one thing. Check comments and fix it.

File: app/views/devise/registrations/edit.html.erb (hunk 1 above)
Comment (SrinandhiniM) [reply app/views/devise/registrations/edit.html.erb:30]: This view files are automatically generated when devise gem is used for sign up and login page.So, we didnt refactor it Magesh

Comment (iMagesh) [reply app/views/devise/registrations/edit.html.erb:30]: yeah but can you refactor it now?

Comment (SrinandhiniM) [reply app/views/devise/registrations/edit.html.erb:30]: ok magesh

**Pull request merged**